src/components/marker-table/index.js

```diff
@@ -231,9 +231,7 @@ export default explicitConnect<{||}, StateProps, DispatchProps>({
     threadsKey: getSelectedThreadsKey(state),
     scrollToSelectionGeneration: getScrollToSelectionGeneration(state),
     getMarker: selectedThreadSelectors.getMarkerGetter(state),
-    markerIndexes: selectedThreadSelectors.getPreviewFilteredMarkerIndexes(
-      state
-    ),
+    markerIndexes: selectedThreadSelectors.getMarkerTableMarkerIndexes(state),
```

Comment:
I still think that we should allow all of them in the marker table without further filtering, but will follow your opinion :-)

(I've a much weaker opinion for the marker chart)